csharp/Clean.cmd

```diff
@@ -1,3 +1,3 @@
-FOR /F "tokens=*" %%G IN ('DIR /B /AD /S bin') DO RMDIR /S /Q "%%G"
```

Comment:
Can you explain what's changing here? Also consider adding REM comments to the batch script for folks (like me) who find it hard to read & understand the script